internal/tsconfig/nextjs.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Next.js",
+  "extends": "./base.json",
+  "compilerOptions": {
```

Comment:
I see some of the `compilerOptions` we are setting explicitly here are already defined in the tsconfig that we are extending. Suggest that we remove any redundant definitions so that it's more clear exactly what we're trying to override from the default that we're extending.